test/integration/parachain/release/redeem.test.ts

```diff
@@ -86,56 +121,93 @@ describe("redeem", () => {
                 BITCOIN_CORE_PORT,
                 `vault_to_liquidate-${collateralCurrency.ticker}-${wrappedCurrency.ticker}`
             );
+
             // Steal some bitcoin (spend from the vault's account)
             const foreignBitcoinAddress = "bcrt1qefxeckts7tkgz7uach9dnwer4qz5nyehl4sjcc";
             const amountToSteal = newMonetaryAmount(0.00001, wrappedCurrency, true);
-            const btcTxId = await vaultBitcoinCoreClient.sendToAddress(foreignBitcoinAddress, amountToSteal);
-            // it takes about 15 mins for the theft to be reported
-            await DefaultTransactionAPI.waitForEvent(api, api.events.relay.VaultTheft, 17 * 60000);
+            const {txid: btcTxId} = await vaultBitcoinCoreClient.sendBtcTxAndMine(foreignBitcoinAddress, amountToSteal, 3);
+            // wait for tx inclusion and block finalization.
+            await waitForBlockFinalization(bitcoinCoreClient, userInterBtcAPI.btcRelay);
+
+            // report theft
+            await reporterInterBtcAPI.vaults.reportVaultTheft(vaultToLiquidateId, btcTxId);
```

Comment:
Nit: Anyone can report theft, so you could have reused `userInterBtcAPI` instead of instantiating `reporterInterBtcAPI`